Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManager.cs

```diff
@@ -246,40 +243,36 @@ public override void Disable()
         {
             base.Disable();
 
-            SettingsProfile.OnCustomHandMaterialUpdate -= UpdateHandMaterial;
+            //SettingsProfile.OnCustomHandMaterialUpdate -= UpdateHandMaterial;
         }
 
         #region Hand Utilities
         protected void UpdateHands()
         {
-            UpdateHand(leftHand, leftSkeleton, leftMeshRenderer, Handedness.Left);
-            UpdateHand(rightHand, rightSkeleton, rightMeshRenderer, Handedness.Right);
+            UpdateHand(rightHand, rightSkeleton, Handedness.Right);
+            UpdateHand(leftHand, leftSkeleton, Handedness.Left);
         }
 
-        protected void UpdateHand(OVRHand ovrHand, OVRSkeleton ovrSkeleton, OVRMeshRenderer ovrMeshRenderer, Handedness handedness)
+        protected void UpdateHand(OVRHand ovrHand, OVRSkeleton ovrSkeleton, Handedness handedness)
         {
-            // Until the ovrMeshRenderer is initialized we do nothing with the hand
-            // This is a bit of a hack because the Oculus Integration fails if we touch the renderer before it has initialized itself
-            if (ovrMeshRenderer == null || !ovrMeshRenderer.IsInitialized) return;
-
             if (ovrHand.IsTracked)
             {
                 var hand = GetOrAddHand(handedness, ovrHand);
-                hand.UpdateController(ovrHand, ovrSkeleton, ovrMeshRenderer, cameraRig.trackingSpace);
+                hand.UpdateController(ovrHand, ovrSkeleton, cameraRig.trackingSpace);
             }
             else
             {
                 RemoveHandDevice(handedness);
             }
         }
 
-        private void UpdateHandMaterial()
-        {
-            foreach (var hand in trackedHands.Values)
-            {
-                hand.UpdateHandMaterial(SettingsProfile.CustomHandMaterial);
-            }
-        }
+        //private void UpdateHandMaterial()
+        //{
+        //    foreach (var hand in trackedHands.Values)
+        //    {
+        //        hand.UpdateHandMaterial(SettingsProfile.CustomHandMaterial);
+        //    }
+        //}
```

Comment:
> Works great with Leap:
> ![LeapRiggedHands](https://user-images.githubusercontent.com/53493796/99434281-71540e80-28c3-11eb-9abe-9a5a9bc48471.gif)
> 
> It would be helpful to add a scene description panel to the RiggedHandsVisualizer example scene, like our other example scenes:
> ![image](https://user-images.githubusercontent.com/53493796/99434651-e1629480-28c3-11eb-8043-62969fe82462.png)
> 
> Also since this is a graduation PR, we would need to move the feature out of the experimental folders, and make sure the namespaces do not include experimental.

@grayclee Do these look ok to you? The thumb sticks out kind of oddly and is more noticeable here than on the quest.